components/service/glean/src/main/java/mozilla/components/service/glean/testing/GleanTestRule.kt

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.testing
+
+import android.content.Context
+import androidx.work.testing.WorkManagerTestInitHelper
+import mozilla.components.service.glean.Glean
+import mozilla.components.service.glean.config.Configuration
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class GleanTestRule(
+    val context: Context
+) : TestRule {
+    override fun apply(base: Statement?, description: Description?): Statement {
+        return object : Statement() {
+            override fun evaluate() {
+                // We're using the WorkManager in a bunch of places, and Glean will crash
+                // in tests without this line. Let's simply put it here.
+                WorkManagerTestInitHelper.initializeTestWorkManager(context)
+
+                Glean.resetGlean(
+                    context = context,
+                    config = Configuration(),
+                    clearStores = true
+                )
+            }
```

Comment:
Aren't you required to execute `base.evaluate()` here to actually run the tests? 🤔

Reply:
Whoops. That explains why my tests were passing so quickly :-| Fixed this, thanks for catching it.